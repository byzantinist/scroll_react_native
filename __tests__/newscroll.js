import 'react-native';
import React from 'react';
import NewScroll from '../components/newscroll';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <NewScroll />
  );
  expect(tree).toMatchSnapshot();
});

