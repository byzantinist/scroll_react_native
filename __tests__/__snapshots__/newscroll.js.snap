// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "height": 1334,
    }
  }
  virtual={undefined}
>
  <View>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#F8F8F8",
            "borderBottomColor": "#a7a6ab",
            "borderBottomWidth": 0.5,
            "elevation": 3,
            "flexDirection": "row",
            "height": 64,
            "justifyContent": "center",
            "left": 0,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 15,
            "right": 0,
            "shadowColor": undefined,
            "shadowOffset": undefined,
            "shadowOpacity": undefined,
            "shadowRadius": undefined,
            "top": 0,
          },
          Object {
            "backgroundColor": "#36485f",
            "borderBottomWidth": 0,
          },
        ]
      }
      virtual={undefined}
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        collapsable={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "#36485f",
            "opacity": 1,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "fontSize": 35,
            }
          }
        >
           🐢
        </Text>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "height": 1334,
          }
        }
        virtual={undefined}
      />
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        collapsable={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "#36485f",
            "opacity": 1,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "fontSize": 35,
            }
          }
        >
          🐇 
        </Text>
      </View>
    </View>
  </View>
  <RCTScrollView
    style={
      Object {
        "backgroundColor": "#36485f",
        "flex": 1,
      }
    }
  >
    <View>
      <View
        collapsable={undefined}
        style={
          Object {
            "backgroundColor": "#dee4ed",
            "marginTop": 476.42857142857144,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
            ],
            "width": 750,
          }
        }
      />
    </View>
  </RCTScrollView>
</View>
`;
